### PR TITLE
unittest-cpp: homepage/url move

### LINKED ISSUE #36400: SHA1 mismatch for unittest-cpp
$ sudo brew install unittest-cpp
==> Downloading https://downloads.sourceforge.net/project/unittest-cpp/UnitTest++/1.4/unittest-cpp-1.4.zip
######################################################################## 100.0%
Error: SHA1 mismatch
Expected: dad944159e2e135aea74039987490eaaee00f2ad
Actual: 557d4c4b3d420911b16c1c477e68403bcebbb717
Archive: /Library/Caches/Homebrew/unittest-cpp-1.4.zip
To retry an incomplete download, remove the file above.
$

### PR DESCRIPTION
**Pull request by DomT4**
The Homepage and URL have moved from SourceForge to Github. The latter is a “fork” maintained by one of the original SourceForge authors and a new maintainer.

Closes #36400

### CODE REVIEW

File: Library/Formula/unittest-cpp.rb [hunk 1]
```diff
@@ -1,21 +1,44 @@
...
 
-    # Install the compiled library
-    lib.install 'libUnitTest++.a'
+      # Install the compiled library
+      lib.install "libUnitTest++.a"
 
-    # Install the documentation
-    doc.install 'docs/UnitTest++.html'
+      # Install the documentation
+      doc.install "docs/UnitTest++.html"
```
Comment (mikemcquaid) [inline Library/Formula/unittest-cpp.rb]: No `make install` outside head?

Comment (DomT4) [inline Library/Formula/unittest-cpp.rb]: Sadly not.

Comment (DomT4) [inline Library/Formula/unittest-cpp.rb]: We didn't have one in the existing formula either, and I think they just pulled the Sourceforge svn repo into Github and then generated a tarball from the contents at the time of the release, so nothing actually changed about the contents, just how and when the package was generated.

File: Library/Formula/unittest-cpp.rb [hunk 2]
```diff
@@ -1,21 +1,44 @@
...
+
+  stable do
+    url "https://github.com/unittest-cpp/unittest-cpp/archive/v1.4.tar.gz"
+    sha1 "ec7bdbebeb6f4d7a069f1125f7b4f473198e491d"
+
+    # Clang failure fixed in the HEAD already
+    fails_with :clang do
+      build 600
+      cause "Failure in test: Expected 2 but was 0"
+    end
```
Comment (mikemcquaid) [inline Library/Formula/unittest-cpp.rb]: Is there a patch we can apply here?

Comment (DomT4) [inline Library/Formula/unittest-cpp.rb]: Ugh. Nope, Sorry. The original patch wouldn't apply cleanly, and then I rebased it and it still wouldn't apply cleanly, so I settled for inreplacing the failed elements and even with the inreplace there the stable build still failed with Clang :anguished:

Comment (DomT4) [inline Library/Formula/unittest-cpp.rb]: It's possible I just suck.

**Pull request closed without merging**